plugins/modules/packaging/os/pacman.py

```diff
@@ -104,6 +104,22 @@
         default:
         type: str
 
+    reason:
+        description:
+            - The install reason to set for the packages
+        choices: [ dependency, explicit ]
+        type: str
+        version_added: 5.4.0
+
+    reason_for:
+        description:
+            - Set the install reason for C(all) packages or only for C(new) packages
+            - In case of C(state=latest) already installed packages which will be updated to a newer version are not counted as C(new)
```

Comment:
```suggestion
            - In case of C(state=latest) already installed packages which will be updated to a newer version are not counted as C(new).
```